readthedocs/doc_builder/backends/sphinx.py

```diff
@@ -632,40 +628,41 @@ def _build_pdflatex(self, tex_files, latex_cwd):
             pdf_commands.append(cmd_ret)
         return all(cmd.successful for cmd in pdf_commands)
 
-    def move(self, **__):
-        if not os.path.exists(self.target):
-            os.makedirs(self.target)
+    def _post_build(self):
+        """Internal post build to cleanup PDF output directory and leave only one .pdf file."""
 
-        exact = os.path.join(
-            self.old_artifact_path,
-            '{}.pdf'.format(self.project.slug),
-        )
-        exact_upper = os.path.join(
-            self.old_artifact_path,
-            '{}.pdf'.format(self.project.slug.capitalize()),
+        if not self.pdf_file_name:
+            log.info("PDF file was not generated/found.")
```

Comment:
This seems odd again to not raise this to the users attention?

Reply:
Yeap. You are correct here. I added the code to raise an exception in this case and also to log a warning if there are multiple PDF generated (following the same idea than for the ePubs)